components/colorloupe/index.css

```
@@ -1,53 +1,97 @@
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
 .spectrum-ColorLoupe {
-  /* size is the same on desktop and mobile */
-  --spectrum-colorloupe-width-adjusted: calc(
-    var(--spectrum-colorloupe-width) +
-      var(--spectrum-global-dimension-static-size-100) /* room for shadow */
-  );
-  --spectrum-colorloupe-height-adjusted: calc(
-    var(--spectrum-colorloupe-height) +
-      var(--spectrum-global-dimension-static-size-100) /* room for shadow */
-  );
+  --spectrum-colorloupe-width: var(--spectrum-color-loupe-width);
+  --spectrum-colorloupe-height: var(--spectrum-color-loupe-height);
 
-  /* Distance between the loupe and the handle */
-  --spectrum-colorloupe-offset: var(
-    --spectrum-global-dimension-static-size-200
-  );
+  --spectrum-colorloupe-offset: var(--spectrum-color-loupe-bottom-to-color-handle);
+  --spectrum-colorloupe-animation-distance: 8px; /* TODO: replace with forthcoming animation token */
 
-  /* How much to animate up to the offset distance */
-  --spectrum-colorloupe-animation-distance: var(
-    --spectrum-global-dimension-static-size-100
-  );
+  --spectrum-colorloupe-drop-shadow-x: var(--spectrum-drop-shadow-x);
+  --spectrum-colorloupe-drop-shadow-y: var(--spectrum-color-loupe-drop-shadow-y);
+  --spectrum-colorloupe-drop-shadow-blur: var(--spectrum-color-loupe-drop-shadow-blur);
+  --spectrum-colorloupe-drop-shadow-color: var(--spectrum-color-loupe-drop-shadow-color);
+
+  --spectrum-colorloupe-outer-border-width: var(--spectrum-color-loupe-outer-border-width);
+  --spectrum-colorloupe-inner-border-width: 1px; /* TODO: replace with updated token for color-loupe-inner-border-width */
+  --spectrum-colorloupe-outer-border-color: var(--spectrum-color-loupe-outer-border);
+  --spectrum-colorloupe-inner-border-color: var(--spectrum-color-loupe-inner-border);
 }
 
 .spectrum-ColorLoupe {
-  width: var(--spectrum-colorloupe-width-adjusted);
-  height: var(--spectrum-colorloupe-height-adjusted);
+  inline-size: var(--mod-colorloupe-width, var(--spectrum-colorloupe-width));
+  block-size: var(--mod-colorloupe-height, var(--spectrum-colorloupe-height));
 
   position: absolute;
-  transform: translate(0, var(--spectrum-colorloupe-animation-distance));
+  transform: translate(0, var(--mod-colorloupe-animation-distance, var(--spectrum-colorloupe-animation-distance)));
   opacity: 0;
   transform-origin: bottom center;
-  bottom: calc(50% + var(--spectrum-colorloupe-offset));
-  left: calc(50% - calc(var(--spectrum-colorloupe-width-adjusted) / 2));
+  inset-block-end: calc(var(--spectrum-colorhandle-size) + var(--mod-colorloupe-offset, var(--spectrum-colorloupe-offset)));
+  inset-inline: calc(40% - calc(var(--mod-colorloupe-width, var(--spectrum-colorloupe-width)) / 2));
   transition: transform 100ms ease-in-out, opacity 125ms ease-in-out;
```

Curious - should these be leveraging tokens?

```suggestion
  transition: transform 100ms ease-in-out, opacity 125ms ease-in-out;
```

---

I had the same thought about `--spectrum-colorloupe-animation-distance: 8px;` and asked in spectrum_css_verifiation about it. 

https://adobedesign.slack.com/archives/GQ09RCBA5/p1678396366690899?thread_ts=1678388705.495289&cid=GQ09RCBA5 

Sounds like Garth has a project in the works to add tokens for animations.

---

Ah gotcha. Cool cool cool.